workflow-testing/src/test/java/com/squareup/workflow1/testing/RealRenderTesterTest.kt

```diff
@@ -1025,6 +1026,100 @@ internal class RealRenderTesterTest {
     }
   }
 
+  @Test fun `verifyActionOutput throws when no output`() {
+    val workflow = Workflow.stateless<Unit, Nothing, Sink<TestAction>> {
+      actionSink.contraMap { it }
+    }
+    val testResult = workflow.testRender(Unit)
+      .render()
+    assertFails {
+      testResult.verifyActionOutput { }
+    }
+  }
+
+  @Test fun `verifyActionOutput provides given output`() {
+    Workflow.stateless<Unit, Int, Sink<TestAction>> {
+      actionSink.send(action { setOutput(1) })
+      actionSink.contraMap { it }
+    }
+      .testRender(Unit)
+      .render()
+      .verifyActionOutput { assertEquals(1, it) }
+  }
+
+  @Test fun `verifyNoActionOutput throws given an output`() {
+    val workflow = Workflow.stateless<Unit, Int, Sink<TestAction>> {
+      actionSink.send(action { setOutput(1) })
+      actionSink.contraMap { it }
+    }
+    val testRender = workflow.testRender(Unit)
+      .render()
+    assertFails {
```

Comment:
Same thing here to tighten up the test

Reply:
I even found a small bug updating this test, it was logging the output wrapper when a value was present instead of the output value directly.